src/com/github/miachm/sods/Style.java

```diff
@@ -14,13 +14,25 @@
     private Color fontColor;
     private Color backgroundColor;
     private int fontSize = -1;
+    private boolean border = false;
+    private boolean borderTop = false;
+    private boolean borderBottom = false;
+    private boolean borderLeft = false;
+    private boolean borderRight = false;
```

Comment:
I don't get why you need the "border" variable. It can be inferred just by checking the other four variables (borderTop, borderBottom, borderLeft, borderRight).